trace_context/compliance.md

```diff
@@ -0,0 +1,19 @@
+# Privacy
+
+Requirements to propagate headers to downstream services opens a potential privacy concerns. The only way to remove this concern is to inspect and remove values from the fields values before allowing the platform or tracing system to execute code that potentially can propagate these headers. All mutations should, however, conform to the list of mutations defined in this specification.
+
+## Privacy of traceparent field
```

Comment:
Not necessary a privacy concern, more like an usability concern is when the service initiating calls use the same trace-parent every time then this cause the service owner to not be able to distinguish between different calls. this my be solved by restarting the trace on the service side if the client is untrusted.

Reply:
Yes. I'd start security section for this. This doc purpose was to ensure the `traceparent` and `tracestate` can be stored and transmitted without implications for privacy and things like GDPR.